src/pages/projects/forms/NetworkSelector.tsx

```diff
@@ -26,34 +28,77 @@ const NetworkSelector: FC<Props & SelectProps> = ({
   const getNetworkOptions = () => {
     const options = managedNetworks.map((network) => {
       return {
-        label: network.name,
+        label: (
+          <div className="label">
+            <span title={network.name} className="network-option u-truncate">
+              {network.name}
+            </span>
+            <span title={network.type} className="network-option u-truncate">
+              {network.type}
+            </span>
+            <span
+              title="network ACLs"
+              className="network-option u-truncate u-align--right"
+            >
+              {getNetworkAcls(network).length || "-"}
+            </span>
```

Comment:
I think it bit more spacing on the right would be nice so the numbers don't sit next to the end. Or maybe not right aligning. Though right alignment is usually good on numbers, but this one probably can be assumed to be single digit.